test/JDBC/expected/test_windows_login-vu-prepare.out

```diff
@@ -0,0 +1,22 @@
+CREATE DATABASE ad_db;
+GO
+
+CREATE LOGIN [ad\Aduser] from windows;
+GO
+
+CREATE LOGIN [ad\Aduserdb] from windows with default_database=[ad_db];
```

Comment:
Let's add verification tests as well by querying sys.server_principal. Or we can even directly query sys.babelfish_authid_login_ext table.

Reply:
Done!